CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 * [#6725](https://github.com/rubocop-hq/rubocop/issues/6725): Mark `Style/SymbolProc` as unsafe for auto-correct. ([@drenmi][])
 * [#6708](https://github.com/rubocop-hq/rubocop/issues/6708): Make `Style/CommentedKeyword` allow the `:yields:` RDoc comment. ([@bquorning][])
 * [#6749](https://github.com/rubocop-hq/rubocop/pull/6749): Make some cops aware of safe navigation operator. ([@hoshinotsuyoshi][])
+* [#6733](https://github.com/rubocop-hq/rubocop/pull/6733): Warn duplicated keys in `.rubocop.yml`. ([@pocke][])
```

Comment:
RuboCop 0.64.0 has been released and I merged #6766. Can you move this entry to `## master (unreleased)` and rebase with the latest master?

Reply:
I've updated the entry of CHANGELOG and rebased, Thanks!

Reply:
It does not seem to be updated. Did you forget force push?

Reply:
Oh, that's right, sorry for my mistake. I pushed it to rubocop-hq/rubocop repo unexpectedly.
I've fixed!